scripts/lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+eslint --config eslint.config.mjs .
```

Comment:
this file is unnecessary. just write a script it package.json and call it from the workflow.